src/fizzBuzz.js

```diff
@@ -0,0 +1,20 @@
+export default function fizzBuzz() {
+  var numbers = []
+  for (var number = 1; number <= 100; number++) {
+
+    if (number % 3 == 0 && number % 5 == 0){
+      numbers.push('FizzBuzz')
+
+    } else if (number % 5 == 0 && number % 3 !== 0) {
```

Comment:
since we already took case of the number being disable by 3 and 5 in the previous if statement, we do not need to explicitly state number % 3 !== 0